pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -222,6 +237,33 @@ func (f *framework) RunPrefilterPlugins(
 	return nil
 }
 
+// RunFilterPlugins runs the set of configured Filter plugins for pod on
+// the given node. If any of these plugins doesn't return "Success", the
+// given node is not suitable for running pod.
+// Meanwhile, the failure message and status are set for the given node.
+func (f *framework) RunFilterPlugins(pc *PluginContext,
+	pod *v1.Pod, nodeName string) *Status {
+
+	_, exist := f.nodeInfoSnapshot.NodeInfoMap[nodeName]
```

Comment:
[podFitsOnNode](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/core/generic_scheduler.go#L492) has used  `f.nodeInfoSnapshot.NodeInfoMap[nodeName]`. If it does not exist, it should return error. So this check seems not needed?

Reply:
You are right and I have removed it from the latest code.